tests/unit/Docker/EnvTest.php

```diff
@@ -37,6 +37,7 @@ public function testExport(): void
         $this->assertEquals("_APP_X=value1
 _APP_Y=value2
 _APP_Z=value3
+_APP_W=value5=
```

Comment:
What is this change?

Reply:
This env variable (_APP_W = value5=)   was added lately  to the resources/docker/.env
inside the test, there is a check against these values.
I can not see how it will pass without this value being added to the test.